versions/2.0/vipps-login-migrate-api-1.0-to-2.0.md

```diff
@@ -0,0 +1,28 @@
+# Vipps Login API
+
+Document version 1.0.0.
```

Comment:
We need to consider what to do with this version number. The current version on master is 3.0.0. If we do not use it we might as well remove it. 

Once we have merged we need to see how we create links between v1 and v2 of the documentation. When we are finalising the documentation we also need to check how it will show up here: https://vipps.no/developers-documentation/login/documentation/ and here https://vipps.no/developers-documentation/